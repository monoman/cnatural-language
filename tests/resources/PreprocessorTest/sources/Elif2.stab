#define YYY
#if XXX || (!YYY)
# undef YYY
    #region R1
/*
    # endregion
#else
#    if !YYY
// Block 1
#    elif (!XXX && YYY)
   // Block 2
# else
    // Block 3
    #endif
#endif // comment //
# define ZZZ
