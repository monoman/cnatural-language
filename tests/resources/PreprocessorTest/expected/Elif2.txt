#define YYY
#if expression: false, skipped: true
....#undef YYY
....#region R1
........----
......../*
........----
....#endregion
#end
#else skipped: false
....#if expression: false, skipped: true
........----
........// Block 1
........----
....#end
....#elif expression: true, skipped: false
........----
........   // Block 2
........----
....#end
....#else skipped: true
........----
........    // Block 3
........----
....#end
#end
#define ZZZ
